src/modules/organisations/services/organisations.service.ts

```diff
@@ -85,7 +85,7 @@ export default class OrganisationsService implements OnApplicationBootstrap {
 			schemaName: gqlOrganisation?.schema_name,
 			createdAt: gqlOrganisation?.created_at,
 			updatedAt: gqlOrganisation?.updated_at,
-			sector: gqlOrganisation?.sector,
+			sector: gqlOrganisation?.haorg_organization_type,
```

Comment:
Let's also add this link between sector and haorg_organization_type in the glossary:
https://docs.google.com/spreadsheets/d/1_vGX_pBIlJeUFt7Laoj8l3iur2qw6CtdqDs-sJ1TSL0/edit#gid=0

maybe under a new section: Organisation: represents a content partner company that does not necessarily have a visitor space. These entries come from the organisations api and are re-cached every night.

Every visitor space and every content partner should have a corresponding entry in the organisations table, but the reverse is not always true.